### ISSUE TITLE
[Bug] Chirpstack integration builds incorrect api url

### ISSUE THREAD

**Issue by Cyril-Roques** [thingsboard/thingsboard#5998]
**Describe the bug**
We try to send a downlink message through chirpstack by using a rule chain with an integration downlink node and chirpstack integration. The uplink part (integration and uplink converter) is working fine, but the downlink part is not.

We are unable to use the chirpstack integration with downlink message because the request to chirpstack use a wrong formated url.
According to our logs the url generated is /api/devices/**true**/queue instead of /api/devices/**1234567890ABCDEF**/queue.

**Your Server Environment**
* own setup
  * cloud docker deployment with official Thingsboard PE images
  * ThingsBoard Version 3.3.1PE
  * OS Name and Version docker host is debian buster

**Your Device**
Shared attribute
* Connectivity
  * Integration: chirpstack

**To Reproduce**
Create a chirpstack integration and a downlink converter with this message output:
```
[{
    "contentType": "JSON",
    "data": "{\"deviceQueueItem\":{\"fPort\":56,\"fCnt\":0,\"data\":\"VRIoE00=\"}}",
    "metadata": {
        "fPort": 56,
        "DevEUI": "1234567890ABCDEF"
    }
}]
```
Create an attribute updated rule chain with a downlink integration node to the previously created integration

**Expected behavior**
a call on the chirpstack's API at the following URL: /api/devices/1234567890ABCDEF/queue

**Additional context**
Thingboard trace and chirpstack response:
```
thingsboard-pe-app               | 2022-02-01 09:00:32,700 [tb-core-consumer-52-thread-2537] WARN  o.t.i.h.c.ChirpStackIntegration - Failed to process downLink message
thingsboard-pe-app               | org.springframework.web.client.HttpClientErrorException$BadRequest: 400 Bad Request: [{"error":"json: cannot unmarshal bool into Go value of type uint32","code":3,"message":"json: cannot unmarshal bool into Go value of type uint32","details":[]}]
thingsboard-pe-app               |     at org.springframework.web.client.HttpClientErrorException.create(HttpClientErrorException.java:101)
thingsboard-pe-app               |     at org.springframework.web.client.DefaultResponseErrorHandler.handleError(DefaultResponseErrorHandler.java:184)
thingsboard-pe-app               |     at org.springframework.web.client.DefaultResponseErrorHandler.handleError(DefaultResponseErrorHandler.java:125)
thingsboard-pe-app               |     at org.springframework.web.client.ResponseErrorHandler.handleError(ResponseErrorHandler.java:63)
thingsboard-pe-app               |     at org.springframework.web.client.RestTemplate.handleResponse(RestTemplate.java:780)
thingsboard-pe-app               |     at org.springframework.web.client.RestTemplate.doExecute(RestTemplate.java:738)
thingsboard-pe-app               |     at org.springframework.web.client.RestTemplate.execute(RestTemplate.java:672)
thingsboard-pe-app               |     at org.springframework.web.client.RestTemplate.postForEntity(RestTemplate.java:447)
thingsboard-pe-app               |     at org.thingsboard.integration.http.chirpstack.ChirpStackIntegration.processDownLinkMsg(ChirpStackIntegration.java:158)
thingsboard-pe-app               |     at org.thingsboard.integration.http.chirpstack.ChirpStackIntegration.onDownlinkMsg(ChirpStackIntegration.java:130)
thingsboard-pe-app               |     at org.thingsboard.server.service.integration.DefaultPlatformIntegrationService.onMsg(DefaultPlatformIntegrationService.java:516)
thingsboard-pe-app               |     at org.thingsboard.server.service.integration.DefaultPlatformIntegrationService.onQueueMsg(DefaultPlatformIntegrationService.java:501)
thingsboard-pe-app               |     at org.thingsboard.server.service.queue.DefaultTbCoreConsumerService.forwardToIntegrationService(DefaultTbCoreConsumerService.java:540)
thingsboard-pe-app               |     at org.thingsboard.server.service.queue.DefaultTbCoreConsumerService.lambda$launchMainConsumers$2(DefaultTbCoreConsumerService.java:250)
thingsboard-pe-app               |     at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
thingsboard-pe-app               |     at org.thingsboard.server.service.queue.DefaultTbCoreConsumerService.lambda$launchMainConsumers$3(DefaultTbCoreConsumerService.java:228)
thingsboard-pe-app               |     at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
thingsboard-pe-app               |     at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
thingsboard-pe-app               |     at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
thingsboard-pe-app               |     at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
thingsboard-pe-app               |     at java.base/java.lang.Thread.run(Thread.java:834)
```

**Comment by volodymyr-babak** [comment]
@Cyril-Roques 
this issue was fixed in 3.3.2PE.
Please upgrade your environment to the 3.3.2PE or 3.3.3PE version.

**Comment by Cyril-Roques** [comment]
Hello,

I have upgraded our thingsboard instance, this issue is indeed gone.

Thank you
